fc/jira/tasks.py

```diff
@@ -79,7 +80,10 @@ def _calc_triage_score(imp_part: str, loe_part: str, date_part: str) -> int:
 
         day_diff = (dt_obj - today).days
 
-        dt_score = _get_date_score(day_diff)
+        if day_diff < 0:
+            dt_score = 20
+        else:
```

Comment:
This logic seems better inside the `_get_date_score` function since it is associated with getting the date score.

Reply:
Yeah, I thought about it, but then didn't put it there because I decided that the get date score function was all about matching the key range in the dictionary.  But, I can maybe be swayed.